Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -355,11 +355,10 @@ their equivalents in the testing library:
 | `try XCTUnwrap(x)` | `try #require(x)` |
 | `XCTFail("…")` | `Issue.record("…")` |
 
-To check that two numeric values are equivalent within a specified
-accuracy, as in the analog of 
-[`XCTAssertEqual(_:_:accuracy:_:file:line:)`](https://developer.apple.com/documentation/xctest/3551607-xctassertequal),
-import [swift-numerics](https://github.com/apple/swift-numerics)
-and use the ``isApproximatelyEqual()`` function.
+The testing library doesn’t provide an equivalent of
+[`XCTAssertEqual(_:_:accuracy:_:file:line:)`](https://developer.apple.com/documentation/xctest/3551607-xctassertequal).
+To compare two numeric values within a specified accuracy, 
+use ``isApproximatelyEqual()`` from [swift-numerics](https://github.com/apple/swift-numerics).
```

Comment:
```suggestion
use `isApproximatelyEqual()` from [swift-numerics](https://github.com/apple/swift-numerics).
```

(DocC uses double backticks to refer to symbols in the same package or project.)